### PR TITLE
Log package and model version in the shell

### PR DESCRIPTION
**Pull request by aalexandrov**
*Description of changes:*

Adds functionality for tracking the `Forecaster` and `gluonts` versions in a `train` / `test` shell.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-200/1 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-200/1/index.html

File: src/gluonts/model/estimator.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 from pydantic import ValidationError
 
 # First-party imports
+from gluonts import __version__ as project_version
```
Comment (jaheba) [inline src/gluonts/model/estimator.py]: I think I like

    import gluonts
    gluonts.__version__

more.

File: src/gluonts/shell/serve.py [hunk 2]
```diff
@@ -101,37 +102,99 @@ def load_config(self) -> None:
...
     def stop(self, *args, **kwargs):
         self.application.logger.info('Shutting down GluonTS scoring service')
 
 
+def make_flask_app(predictor_factory, execution_params) -> Flask:
+    app = Flask('GluonTS scoring service')
+
+    @app.route('/ping')
+    def ping() -> Response:
+        return Response(status=200)
```
Comment (jaheba) [inline src/gluonts/shell/serve.py]: Why return the verbose response?

Comment (aalexandrov) [reply src/gluonts/shell/serve.py]: The PyCharm linter was complaining due to the type mismatch. Reverted.

**Comment by szha** [comment]
Job PR-200/3 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-200/3/index.html

**Pull request merged**